charts/apisix/values.yaml

```diff
@@ -157,6 +157,16 @@ apisix:
   #         key: KEY
   extraEnvVars: []
 
+  # -- Set the readinessProbe for Apache APISIX pods
+  readinessProbe:
+    failureThreshold: 6
+    initialDelaySeconds: 10
+    periodSeconds: 10
+    successThreshold: 1
+    tcpSocket:
+      port: &gatewayHttpContainerPort 9080
```

Comment:
This is incorrect.

Reply:
I used `YAML Anchors` to refer to the same values.
Wouldn't you recommend it?
https://helm.sh/docs/chart_template_guide/yaml_techniques/#yaml-anchors.

Reply:
Sorry, this does work. I may have made a mistake during my previous verification.